tensorboard/webapp/metrics/views/card_renderer/scalar_card_fob_controller.ts

```diff
@@ -76,20 +78,18 @@ export class ScalarCardFobController {
       return null;
     }
     return this.scale.forward(
-      this.minMax,
+      this.minMaxHorizontalViewExtend,
       [0, this.axisSize],
       this.timeSelection.end.step
     );
   }
 
   getHighestStep(): number {
-    const minMax = this.minMax;
-    return Math.floor(minMax[0] < minMax[1] ? minMax[1] : minMax[0]);
```

Comment:
Why is this check no longer needed?

Reply:
As stated in the description, the idea of using minMax to get the highest step is not correct.
The old minMax is actually view extend, which could be negative. For example, the step is [0, 2000] and the viewextend is [-200, 2200]  (see the console.log)

![Screen Shot 2022-10-24 at 3 25 18 PM](https://user-images.githubusercontent.com/1131010/197622761-5c5123f2-2afe-47d5-b645-8155af1380e8.png)

I did not look into the math here 
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/widgets/line_chart_v2/line_chart_internal_utils.ts;l=31
but a couple of checks confirms minMax is _not_ min max steps we need.